packages/services/src/providers/perps/symphony-perps.provider.ts

```diff
@@ -237,11 +237,23 @@ export class SymphonyPerpsProvider implements IPerpsDataProvider {
 
   /**
    * Get account summary - transforms Symphony response to generic format
+   * @param walletAddress Wallet address to query
+   * @param initialCapital Optional initial capital for validation (Symphony tracks this internally)
    */
-  async getAccountSummary(walletAddress: string): Promise<PerpsAccountSummary> {
+  async getAccountSummary(
+    walletAddress: string,
+    initialCapital?: number,
+  ): Promise<PerpsAccountSummary> {
     const startTime = Date.now();
     const maskedAddress = this.maskWalletAddress(walletAddress);
 
+    // Log if initial capital is provided (for debugging provider compatibility)
+    if (initialCapital !== undefined) {
+      this.logger.debug(
```

Comment:
maybe add the `walletAddress` here for debugging? although, idk if this log is even needed because it'll be inferred by the next log due to `initialCapital` being part of `criticalFields`

Reply:
Resolved in [85b03b2](https://github.com/recallnet/js-recall/pull/1370/commits/85b03b23af4a778f415c10dd7be06c75335efba4), but technically this is just for historical "correctness" as the initial capital field is not actually used when calculating Calmar (instead, the first and latest snapshots are used). This was added for provider compatibility since the Hyperliquid api does not track this field but Symphony does.